Sources/hostmgr-helper/VirtualMachineSlot.swift

```diff
@@ -87,7 +88,13 @@ class VirtualMachineSlot: NSObject, ObservableObject {
         default:
             break
         }
-        try await managedVirtualMachine?.machine.stop()
+        if let machine = managedVirtualMachine?.machine, machine.canStop {
```

Comment:
All this really means is we're being a little "nicer" to a VM that we're abruptly pulling the plug in anyway. I'm not sure of the different between this and just setting the reference to `nil`, but I presume it would be in how the process exits.